test/socket_test.dart

```diff
@@ -1,28 +1,40 @@
+import 'dart:convert';
 import 'dart:io';
 
+import 'package:mockito/mockito.dart';
+import 'package:realtime_client/src/lib/constants.dart';
 import 'package:test/test.dart';
 import 'package:realtime_client/realtime_client.dart';
 import 'package:web_socket_channel/io.dart';
+import 'package:web_socket_channel/web_socket_channel.dart';
+
+import 'socket_test_stubs.dart';
 
 typedef IOWebSocketChannelClosure = IOWebSocketChannel Function(
     String url, Map<String, String> headers);
 
 void main() {
+  const int int64MaxValue = 9223372036854775807;
+
+  const socketEndpoint = 'wss://localhost:0/';
+
   HttpServer mockServer;
 
   setUp(() async {
     mockServer = await HttpServer.bind('localhost', 0);
     mockServer.transform(WebSocketTransformer()).listen((webSocket) {
       final channel = IOWebSocketChannel(webSocket);
       channel.stream.listen((request) {
-        channel.sink.add(request);
+        // channel.sink.add(request);
```

Comment:
not sure why you comment out this line. There's no echo message back to client ws. the test will fail